cluster-state-service/handler/api/v1/translate.go

```diff
@@ -87,50 +88,55 @@ func toContainerInstanceResource(r *types.Resource) *models.ContainerInstanceRes
 	return resource
 }
 
-// ToContainerInstance tranlates a container instance represented by the internal structure (types.ContainerInstance) to it's external representation (models.ContainerInstance)
-func ToContainerInstance(instance types.ContainerInstance) (models.ContainerInstance, error) {
-	err := validateContainerInstance(instance)
+// ToContainerInstance translates a container instance represented by the internal structure (storetypes.ContainerInstanceWrapper) to it's external representation (models.ContainerInstance)
+func ToContainerInstance(instanceWrapper storetypes.ContainerInstanceWrapper) (models.ContainerInstance, error) {
+	err := validateContainerInstance(instanceWrapper.ContainerInstance)
```

Comment:
I would add `c := instanceWrapper.ContainerInstance` before this and use `c` everywhere to make the code look cleaner.